geometry_msgs/TransformStamped object_pose
float64[] start_state
string planning_group
---
geometry_msgs/TransformStamped pre_grasp_pose
moveit_msgs/RobotTrajectory trajectory
bool success
float32 gripper_width
